### PR TITLE
Fix/Disable stylelint errors for scales/radii

### PR DESCRIPTION
**Pull request by manzoorwanijk**
#### Proposed Changes

See #67467 

- Diable warnings for `scales/radii` rule
- Remove unnecessary `stylelint-disable` comments

#### Testing Instructions

- Do a sanity check on Calypso live link

### CODE REVIEW

File: client/components/backup-storage-space/style.scss [hunk 1]
```diff
@@ -22,20 +22,18 @@
  * .progress-bar__progress: the filled part of the bar
 **/
 
-/* stylelint-disable scales/radii --
-   We need to be able to define a border-radius of 1/2 the parent height
-*/
```
Comment (manzoorwanijk) [inline client/components/backup-storage-space/style.scss:27]: This comment prevented the file from being auto-fixed.

File: .stylelintignore [hunk 2]
```diff
@@ -5,3 +5,5 @@
 node_modules
 vendor
 dist
+packages/calypso-color-schemes/css
+packages/calypso-color-schemes/src/__color-studio
```
Comment (manzoorwanijk) [inline .stylelintignore:9]: Errors for `indentation` are caused for the files which are `.gitignore`ed. So, adding those here will remove those errors.

**Comment by noahtallen** [comment]
Ah, I was a little confused by this at first -- but the idea is that we're changing `stylelint-disable` to the more specific disable rules (like disable-next-line)?

**Review by noahtallen — APPROVED** [review]
LGTM!

**Pull request merged**